BoxContentSDK/BoxContentSDK/Requests/BOXUserRequest.h

```diff
@@ -14,6 +14,10 @@
  */
 @property (nonatomic, readwrite, assign) BOOL requestAllUserFields;
 
+@property (nonatomic, readonly, strong) NSString *userID;
```

Comment:
Should we leave a comment that this property only reports a value if a specific user is requested, and that in the default case where no user is specified the current user is used (but this value will be nil or an empty string)?

Reply:
Good point, for a manual construction of the request, setting the userID will have the same impact as the initWithUserID